states/c172p-water-overlay.xml

```diff
@@ -0,0 +1,94 @@
+<?xml version="1.0"?>
+
+<PropertyList>
+
+  <name type="string">water</name>
+
+  <overlay>
+
+    <sim>
+      <current-view>
+        <view-number type="int">0</view-number>
+      </current-view>
+      <freeze>
+	      <master type="bool">0</master>
+	      <clock type="bool">0</clock>
+      </freeze>
+      <presets>
+	      <speed-set type="string">knots</speed-set>
+	      <airspeed-kt type="int">80</airspeed-kt>
+	      <longitude-deg type="double">-96.785</longitude-deg>
+	      <latitude-deg type="double">37.919</latitude-deg>
+	      <onground type="bool">0</onground>
+	      <altitude-ft type="int">2000</altitude-ft>
+	      <heading-deg type="double">191</heading-deg>
+      </presets>
+      <model>
+	      <c172p>
```

Comment:
These lines here make me sad :see_no_evil: 

```sh
XMLLINT_INDENT="    " xmllint --format the-file --output the-file
```

Add back some blank lines where appropriate.

Reply:
I don't understand this? What are you trying to tell me here?

For one thing, the water-overlay I added for testing convenience while @dany93 and myself were working on this. It will be removed eventually and added when we do the "state"  work.

Reply:
oh, tabs... my bad... can't see them in the diff preview... but you can see that the spacing is off from the other code...

Reply:
Since this file is a new file, it's best to just format it properly with the `xmllint` tool (see above).